aws/rosa-hcp-dual-region/camunda-version/8.7/procedure/camunda/export_environment_prerequisites.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+###############################################################################
+# Important: Adjust the following environment variables to your setup         #
+###############################################################################
+
+# The script must be executed with
+# . ./export_environment_prerequisites.sh
+# to export the environment variables to the current shell
+
+# The AWS regions of your OpenShift cluster 1 and 2
+export CLUSTER_1_REGION=us-east-1
+export CLUSTER_2_REGION=us-east-2
+
+# The names of your OpenShift clusters in regions 1 and 2
+export CLUSTER_1_NAME=""
+export CLUSTER_2_NAME=""
+
+# The OpenShift namespaces for each region where Camunda 8 should be running
+# Namespace names must be unique to route the traffic
+export CAMUNDA_NAMESPACE_1="camunda-$CLUSTER_1_NAME"
+export CAMUNDA_NAMESPACE_2="camunda-$CLUSTER_2_NAME"
+
+# The backup bucket access variables
```

Comment:
as mentioned could link to the related doc part of the aws openshift dual-region setup guide.
Just as an optional assistance for people that follow end2end